go/vt/discovery/healthcheck_test.go

```diff
@@ -370,6 +438,10 @@ func TestHealthCheckTimeout(t *testing.T) {
 		t.Errorf(`<-l.output: %+v; want not serving`, res)
 	}
 
+	if !fc.isCancelled() {
+		t.Errorf("StreamHealth should be cancelled after timeout, but is not")
```

Comment:
nit: "canceled" throughout the file because we're using it everywhere. (Go agreed on that spelling as well.)